src/priv-csrs.adoc

```diff
@@ -438,25 +438,29 @@ Supervisor interrupt pending.
 `0x603` +
 `0x604` +
 `0x606` +
-`0x607` 
+`0x607` +
+`0x612`
 |HRW + 
 HRW +
 HRW +
 HRW +
 HRW +
+HRW +
 HRW 
 |`hstatus` +
 `hedeleg` +
 `hideleg` +  
 `hie` +  
 `hcounteren` + 
-`hgeie`
+`hgeie` +
+`hedelegh`
 |Hypervisor status register. +
 Hypervisor exception delegation register. +
 Hypervisor interrupt delegation register. +
 Hypervisor interrupt-enable register. +
 Hypervisor counter enable. +
-Hypervisor guest external interrupt-enable register.
+Hypervisor guest external interrupt-enable register. +
+Additional hypervisor exception delegation register, RV32 only.
```

Comment:
Change the description of `hedelegh` to be:  "Upper 32 bits of `hedeleg`, RV32 only".